packer/build_image.sh

```diff
@@ -238,12 +226,6 @@ else
         exit 1
     fi
 
-    import_gpg_key
-
-    SCYLLA_FULL_VERSION=$(get_version_from_remote_deb $PRODUCT-server)
-    SCYLLA_MACHINE_IMAGE_VERSION=$(get_version_from_remote_deb $PRODUCT-machine-image)
-    SCYLLA_PYTHON3_VERSION=$(get_version_from_remote_deb $PRODUCT-python3)
-
     sudo rm -f $TMPREPO
```

Comment:
In addition this variable `TMPREPO` is not exist anymore, the creation of this temp file was part of `import_gpg_key` function

Reply:
Removed